battle_abilities_test.go

```diff
@@ -24,3 +24,24 @@ var _ = Describe("Scrappy", func() {
 		Entry("fighting", TypeFighting),
 	)
 })
+
+var _ = Describe("Ability: Iron Fist", func() {
+	a := Agent(new(dumbAgent))
+
+	It("should have punching moves deal 1.2x damage", func() {
+		b := New1v1Battle(
+			GeneratePokemon(PkmnMachop, WithLevel(15), WithAbility(AbilityIronFist), WithMoves(MoveCometPunch)), &a,
+			GeneratePokemon(PkmnBidoof, WithLevel(15), WithMoves(MoveCometPunch)), &a,
+		)
+		b.rng = AlwaysRNG()
```

Comment:
OOC, why not `SimpleRNG()`?

Reply:
So the attacks don't miss

Reply:
The goal of simple RNG was to have a factory that produces attacks that always hit but don't have other side effects (stat changes, crits, etc.). Maybe it is too difficult now to trace this behavior?